### ISSUE TITLE
[subviews] different screen shake timing

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#1137]
More of a note to self to investigate. It's obvious from our demo cutscene, since it doesn't play out properly.

It also uses viewport shaking, so a nice place to recheck #999.

**Comment by lynxlynxlynx** [comment]
Comparing the two, the difference is actually in the shake — it's delayed in subviews.

**Comment by lynxlynxlynx** [comment]
Running the action manually works, but in this cutscene it is delayed until the next scripted viewport move. Trivial to verify by increasing the wait in the script after the shake.

```
        Wait(4)
        ScreenShake([10.10],30)
//      TriggerActivation("Caveentrance",TRUE)
//      OpenDoor("swirldoor")
//      AmbientActivate("swirl",FALSE)
        Wait(10) // bumped
        MoveViewObject(Player1,4)
```

I couldn't just do a lazy bisect, since the demo is too new. Some changes in #999 look suspect, but I don't see anything there that would delay DoStep, so my money is on some older regression. Will find or create a test in one of the games.

**Comment by lynxlynxlynx** [comment]
This has been fixed by the changes for #1138, however there is a new problem — opening a clean report.
